packages/postgress/src/interfaces.ts

```diff
@@ -0,0 +1,15 @@
+import { Logger } from '@credo-ts/core'
+
+export interface ConnectionInfo {
+  sendPushNotification?: (messageId: string) => Promise<void>
+  maxReceiveBytes?: number
```

Comment:
```suggestion
```

Let's remove this to simplify, since we are not using it yet

Reply:
solved!